torch/ao/quantization/fake_quantize.py

```diff
@@ -211,46 +214,25 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
         super(FakeQuantize, self)._load_from_state_dict(state_dict, prefix, local_metadata, strict,
                                                         missing_keys, unexpected_keys, error_msgs)
 
-class FixedQParamsFakeQuantize(FakeQuantizeBase):
+
+class FixedQParamsFakeQuantize(FakeQuantize):
```

Comment:
now that we have `FixedQParamsObserver`, can we remove `FixedQParamsFakeQuantize` (doesn't have to be in this PR)?

Reply:
I had actually removed it in the original changes, but then I realized it's actually a public API and would break BC so I reverted it. But yes it doesn't serve much purpose right now

Reply:
I would vote for breaking BC and removing this. We never officially communicated the presence of this, we can explain the change in the release notes, and the number of affected people should be very low.

Reply:
yeah I think we want to remove it in a separate PR since it's a BC breaking change